<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>solarizeForm</class>
 <widget class="QWidget" name="solarizeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>671</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Limiar de Vermelho</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="inputsolarizeFormLimiarR">
      <property name="cursor">
       <cursorShape>ClosedHandCursor</cursorShape>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Limiar de Verde</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="inputsolarizeFormLimiarG">
      <property name="cursor">
       <cursorShape>ClosedHandCursor</cursorShape>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Limiar de Azul</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="inputsolarizeFormLimiarB">
      <property name="cursor">
       <cursorShape>ClosedHandCursor</cursorShape>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_3">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>701</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Esta função gera o efeito de Solarize na imagem.
O Solarize inverte as cores acima do valor de limiar especificado e mantém as que estiverem abaixo.
Controle o efeito selecionando os limiares de Vermelho,Verde e Azul.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>inputsolarizeFormLimiarR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>inputsolarizeFormLimiarR</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputsolarizeFormLimiarG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>inputsolarizeFormLimiarG</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputsolarizeFormLimiarB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>688</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>inputsolarizeFormLimiarB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
